src/pages/_layout.svelte

```diff
@@ -71,6 +71,8 @@
     overflow: hidden;
     max-width: 155rem;
     margin: 0 auto;
+    display: flex;
+    flex-direction: column;
```

Comment:
Have you tested whether this doesn't have undesirable effects on any other pages?
I'd rather you make a new container on the 404 page than style that, rather than make every other page `flex`